tagstudio/src/core/media_types.py

```diff
@@ -145,7 +151,14 @@ class MediaCategories:
         ".sqlite",
         ".sqlite3",
     }
-    _DISK_IMAGE_SET: set[str] = {".bios", ".dmg", ".iso"}
+    _DISK_IMAGE_SET: set[str] = {
+        ".bios",
+        ".dmg",
+        ".iso",
+        # ".img", ##Legacy Mac Disk Image/Floppy Disk Image
+        # ".vmdk" ##VM DisK image
+        # ".dmg" ##Apple Disk Image
```

Comment:
.dmg is already in the set. Please only use one leading hash for comments (ruff E266)